packages/components-js/tests/vrt/playwright/specs/custom/popover.vrt.ts

```diff
@@ -15,7 +15,7 @@ const scenario = async (page: Page, theme: Theme, scheme?: PrefersColorScheme):
   };
 
   const markup =
-    () => `<div style="position: relative; height: 800px; outline: 1rem solid rgba(0, 0, 255, 0.1); outline-offset: -1rem">
+    () => `<div style="position: relative; height: 800px; outline: 1rem solid rgba(0, 0, 255, 0.1); outline-offset: -1rem" class="playground">
```

Comment:
I think the playground class isn't necessary in this context?

Reply:
Necessary for `waitForForcedComponentTheme` to set the theme class